apps/discord-bot/src/commands/minecraft/available.command.ts

```diff
@@ -0,0 +1,44 @@
+import { MojangPlayerArgument } from '#arguments';
+import { SUCCESS_COLOR, WARNING_COLOR } from '#constants';
+import { MojangApiService } from '#services';
+import { Command, CommandContext, EmbedBuilder } from '@statsify/discord';
+
+@Command({ description: 'commands.available', args: [new MojangPlayerArgument()] })
+export class AvailableCommand {
+  public constructor(private readonly mojangApiService: MojangApiService) {}
+
+  public async run(context: CommandContext) {
+    const name = context.option<string>('player');
+
+    const nameInfo = await this.mojangApiService.checkName(name.trim());
+
+    if (!nameInfo) {
+      const embed = new EmbedBuilder()
+        .field((t) => t('embeds.available.description.username'), name)
+        .field(
+          (t) => `*${t('embeds.available.description.status')}`,
+          (t) => t('generic.available')
```

Comment:
```suggestion
          (t) => t('available')
```

You don't actually have a generic field in the lang file